resources/lib/database/db_base_sqlite.py

```diff
@@ -28,6 +28,19 @@
 # LIKE comparator use ASCII, the unicode chars are not comparable
 # ---------------------------------------------------------------------------
 
+def is_libsqlite3_threadsafe():
```

Comment:
this method since it not depends from nothing can be moved in to 
db_utils.py and please rename as `is_sqlite3_threadsafe`

at same time the `global` variables can be full removed,
so in globals.py you can add in to `init_database` method as first line:
`self.IS_SQLITE3_THREADSAFE = is_sqlite3_threadsafe()`
^^ method imported locally from db utils